### DISCUSSION TITLE
Customizing Admin Panel for Vendors in Marketplace v2

### DISCUSSION THREAD

**Discussion by chrislicodes** [medusajs/medusa#9458] [Q&A]
Hi everyone,

I'm currently working through the [Marketplace v2](https://docs.medusajs.com/v2/resources/recipes/marketplace/examples/vendors) example, and I had a question regarding customization.

Towards the end of the documentation, it mentions:

> If your use case requires bigger customizations to the admin, such as showing different products and orders based on the
> logged-in vendor, use the [admin API routes](https://docs.medusajs.com/v2/api/admin) to build a custom admin.

My question is: where exactly would I implement these customizations?

In the [Medusa Eats](https://github.com/medusajs/medusa-eats) example, it looks like each user type has their own dedicated view in the store-front to interact with the backend. I also came across [Mercur JS](https://www.npmjs.com/package/mercurjs), which seems to fork the Admin UI and render it on a separate /vendor UI on the server. 

The default Admin Panel is fantastic, so the vendor might just need a streamlined version of it with fewer features. Is there a recommended approach for doing this? Or are there any plans to release a UI example or best practices for vendor-specific admin views?

Thanks in advance for any guidance!

**Comment by okechukwu0127** [comment]
I am really hopping for an answer to this.
After setting up the Marketplace receipe, i got lost on what to do next as I am stock with building the fresh Admin UI for the Vendor to manage product and the entire order workflow which is a huge task to pull off.

Still wondering why create a vendor/vendor-admin account that cant login to the current Admin-UI of Medusa 2

**Comment by okechukwu0127** [comment]
> Hi everyone,
> 
> I'm currently working through the [Marketplace v2](https://docs.medusajs.com/v2/resources/recipes/marketplace/examples/vendors) example, and I had a question regarding customization.
> 
> Towards the end of the documentation, it mentions:
> 
> > If your use case requires bigger customizations to the admin, such as showing different products and orders based on the
> > logged-in vendor, use the [admin API routes](https://docs.medusajs.com/v2/api/admin) to build a custom admin.
> 
> My question is: where exactly would I implement these customizations?
> 
> In the [Medusa Eats](https://github.com/medusajs/medusa-eats) example, it looks like each user type has their own dedicated view in the store-front to interact with the backend. I also came across [Mercur JS](https://www.npmjs.com/package/mercurjs), which seems to fork the Admin UI and render it on a separate /vendor UI on the server.
> 
> The default Admin Panel is fantastic, so the vendor might just need a streamlined version of it with fewer features. Is there a recommended approach for doing this? Or are there any plans to release a UI example or best practices for vendor-specific admin views?
> 
> Thanks in advance for any guidance!

Cant wait to find a way to log the Vendor/ Vendor-admin into the Medusa 2 admin-UI

**Reply by RajnandiniIngale** [reply]
@okechukwu0127 did you get the solution as I also wanted to know on how to log the vendor-admin to the medusa 2 admin-ui

**Reply by okechukwu0127** [reply]
try this out

https://medium.com/@igorkhomenko/building-a-multivendor-marketplace-with-medusa-js-2-0-a-dev-guide-f55aec971126

**Reply by okechukwu0127** [reply]
The 2nd part of the series about Super Admin functionality is available here 
https://medium.com/@igorkhomenko/building-a-multivendor-marketplace-with-medusa-js-2-0-super-admin-d899353b0b1e
